src/qt/bitcoingui.h

```diff
@@ -143,6 +147,10 @@ class BitcoinGUI : public QMainWindow
     HelpMessageDialog* helpMessageDialog = nullptr;
     ModalOverlay* modalOverlay = nullptr;
 
+#ifdef Q_OS_MAC
+    CAppNapInhibitor* appNapInhibitor = nullptr;
```

Comment:
Should be deleted? nit, `m_app_nap_inhibitor`;

Reply:
Delete what and why? 🤔